modules/admin_manual/pages/enterprise/logging/admin_audit.adoc

```diff
@@ -31,7 +31,7 @@ image:enterprise/logging/admin_auditing.png[Auditing]
 
 == Configuration
 
-It is advised to redirect messages into a separate file. To do so, add these lines to `config.php` and adjust the target path accordingly. Note that the target path must be writeable for the web server user:
+It is advised to redirect messages into a separate file. To do so, add these lines to `config.php` and adjust the target path accordingly. Otherwise please make sure to adjust the server log level to 1 (or 0), otherwise the Auditing app won't log anything. Configuring a seperate file circumvents the global log level. Note that the target path must be writeable for the web server user:
```

Comment:
```suggestion
It is advised to redirect messages into a separate file. To do so, add these lines to `config.php` and adjust the target path accordingly. Otherwise make sure to adjust the server log level to 1 (or 0), or else the Auditing app won't log anything. Configuring a separate file circumvents the global log level. Note that the target path must be writable for the web server user:
```